auto-actions-story-status-controller

From: Bryan Larsen <bryan@larsen.st>

Whenever you create a new model + controller with Hobo, get into the habit of thinking about permissions and controller actions. In this case, we probably want only admins to be able to manage the permissions. As for actions, we probably only want the write actions, and the index page:

SHOW_PATCH
---
 app/controllers/story_statuses_controller.rb |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/app/controllers/story_statuses_controller.rb b/app/controllers/story_statuses_controller.rb
index 53a3817..318bad5 100644
--- a/app/controllers/story_statuses_controller.rb
+++ b/app/controllers/story_statuses_controller.rb
@@ -2,6 +2,6 @@ class StoryStatusesController < ApplicationController
 
   hobo_model_controller
 
-  auto_actions :all
+  auto_actions :write_only, :new, :index
 
 end
